eng/native/functions.cmake

```diff
@@ -268,7 +268,7 @@ function(strip_symbols targetName outputFilename)
 
       string(TOLOWER "${CMAKE_BUILD_TYPE}" LOWERCASE_CMAKE_BUILD_TYPE)
       if (LOWERCASE_CMAKE_BUILD_TYPE STREQUAL release)
-        set(strip_command ${STRIP} -no_code_signature_warning -S ${strip_source_file})
+        set(strip_command ${STRIP} -no_code_signature_warning -S ${strip_source_file} && codesign -f -s - ${strip_source_file})
```

Comment:
Is the `-f` option needed? I've been using the same change as you have here locally without the -f and I haven't hit any issues.

Reply:
On the DTK, if there was a corrupted signature the `codesign -s -` command failed without either
- Adding the `-f` 
- running `codesign --remove-signature`

Reply:
It might have because the corrupt adhoc signature was from a different machine & user